decoders/avr_isp/pd.py

```diff
@@ -223,6 +242,10 @@ def handle_command(self, cmd, ret):
             self.handle_cmd_read_fuse_high_bits(cmd, ret)
         elif cmd[:3] == [0x50, 0x08, 0x00]:
             self.handle_cmd_read_extended_fuse_bits(cmd, ret)
+        elif cmd[:3] == [0x38, 0x00, 0x00]:
```

Comment:
I'm not sure I understand how this set of if/elif is sorted (by... alphabetical function name ?) but it doesn't look like your additions quite fit where they should

Reply:
Added a commit that sorts the 'elif cmd[:3]' based on the 3-byte value.